typhos/tests/conftest.py

```diff
@@ -197,14 +204,11 @@ def pytest_runtest_call(item: pytest.Item):
         else:
             logger.error(failure_text)
 
-    try:
-        assert not final_widgets, failure_text
```

Comment:
This is the main thing I want to re-evaluate before asking for review. I'll fix up the desc before requesting a review too.

Reply:
I removed this edit because I thought it was a somewhat important check and it passed offline in all permutations.
Then, it failed again here because pyqtgraph isn't being cleaned up in a timely manner (or at all?) in the online CI only.

Reply:
I'm going to add another mark, apply it to the failing function, and then move on.